main.scss

```diff
@@ -66,8 +66,10 @@
 
 	// Sizes.
 	@each $size in $sizes {
-		.o-buttons--#{$size} {
-			@include _oButtonsSizeContent($size);
+		@if index($_o-buttons-sizes, $size) {
```

Comment:
To confirm I understand this correctly. Currently we would output a class here for any given value in `$size` and with this change we would only output classes where `$size` matches an entry in `$_o-buttons-sizes`?

Reply:
Correct